index.bs

```diff
@@ -7992,9 +7989,9 @@ a reference to the same object that the IDL value represents.
         starting at byte offset |offset|.
 </div>
 
-<div algorithm="detach an arraybuffer">
+<div algorithm>
```

Comment:
Can I ask what's the difference between `algorithm="something"` and just `algorithm`?